docs/development-environment.md

```diff
@@ -257,6 +257,10 @@ You can read the details in [docker/README.md](../docker/README.md)
 
 You should do all Jetpack development with `define( 'WP_DEBUG', true );` in your `wp-config.php`, making sure that you’re not generating any Notices or other PHP issues in your error_log.
 
+### SCRIPT_DEBUG
+
+By default, WordPress loads minified versions of Jetpack's JS files. If you want to work with them, add `define( 'SCRIPT_DEBUG', true );` in your `wp-config.php`. This tells WordPress to load the non-minified JS version, allowing you to see your changes on page refresh.
```

Comment:
I wonder if we should specify that this is about the module files (outside of `_inc/client/` and `extensions/`). That may make things a bit clearer to new contributors?

Reply:
Yes, I'll add that. Actually I thought it was only for the js files in the modules folder but I'm not sure about that...

Reply:
It is mostly files inside `modules/`, yes, but it also includes a few standalone js files outside of that directly, and in `_inc/` for example.

Reply:
@jeherve I've added the extra info :)